class_generator/README.md

```diff
@@ -22,6 +22,12 @@ if type class-generator > /dev/null; then eval "$(_CLASS_GENERATOR_COMPLETE=zsh_
 
 ###### Call the script
 
+- Calling the script with poetry
+
+```bash
+poetry run class-generator --kind <kind>
```

Comment:
This is not needed, once running poetry install it will install it system-wide and run it inside poetry.

Reply:
What is needed is to change the install command, installation should be done with pipx or pip

using pipx
pipx install openshift-python-wrapper

using pip
python 3 -m pip install openshift-python-wrapper

Reply:
ack I